.travis.yml

```diff
@@ -1,14 +1,17 @@
 language: scala
 jdk: oraclejdk8
 scala: 2.12.8
+before_install: curl -Lo coursier https://git.io/coursier-cli && chmod +x coursier
 
 # make sure to fetch enough commits, so that git describe still works
 git:
   depth: false
 
 jobs:
   include:
-    - script: sbt test headerCheck test:headerCheck
+    - script:
+      - ./coursier launch org.scalameta:scalafmt-cli_2.12:2.0.0-RC4 -- --test
+      - sbt test headerCheck test:headerCheck
```

Comment:
Why not simply use `sbt scalafmtCheck`?

Reply:
it also have the additional value that the scalafmt version will be the same as declated in plugins.sbt

Reply:
There is no version of scalafmt in plugins.sbt because there is no sbt-scalafmt that supports scalafmt 2.0.0-RC4 yet.  Also, using scalafmt at the command-line (and in the IDE) is the officially recommended way to use scalafmt (which is also why there is no sbt-scalafmt for 2.0.0-RC4).

So this is the way I'm experimenting with.  I could move launching scalafmt to a shell script, maybe.

Reply:
Note that, when there is a release of sbt-scalafmt that supports scalafmt 2.0.0-RC4, I'm ok with switching to it.  I actually was the one that opened the tracking issue for it 😄

https://github.com/scalameta/sbt-scalafmt/issues/15